packages/parser/test/Render_test.t

```diff
@@ -0,0 +1,4 @@
+  $ alias run='./Render_test.exe'
+
+  $ cat ./test.css | run
+  .some-grid{grid:[line1] minmax(20em, max-content) / auto-flow dense 40%;}.a,href > h1,h3[data-foo*=bar]{&::after,href > h1,&:is(h1),&:has(+ :is(h3)){padding:20px 30px calc(20px+30px*(20 * 400/2));}background-image:radial-gradient(circle, #0000 45%, #000f 48%), radial-gradient(ellipse farthest-corner, #fc1c14 20%, #cf15cf 80%);grid-template-columns:[line-name1] 100px [line-name2] repeat(auto-fit, [line-name3 line-name4] 300px) ;}abbr{color:chocolate;}@media (hover: hover) {abbr:hover{color:limegreen;transition-duration:1s;}}@media not all and (hover: hover) {abbr::after{content:" (" attr(title) ")";}}
```

Comment:
Can we have those tests as "idempotent" and parse-print-parse and compare?

I would prefer to keep separate "features" as separate tests, but a big one is also cool

Reply:
maybe the "big" one can be a small tailwind generated file?

Reply:
yup, will be adding the parse-print-parse test asap

Reply:
> maybe the "big" one can be a small tailwind generated file?

yup great idea